src/dodal/devices/turbo_slit.py

```diff
@@ -1,8 +1,9 @@
-from ophyd_async.core import StandardReadable
+from bluesky.protocols import Movable
+from ophyd_async.core import AsyncStatus, StandardReadable
 from ophyd_async.epics.motor import Motor
 
 
-class TurboSlit(StandardReadable):
+class TurboSlit(StandardReadable, Movable):
```

Comment:
```suggestion
class TurboSlit(StandardReadable, Movable[float]):
```

Now that bluesky 1.13.1 is in again, should reflect that

Reply:
But also, the set method just defers to a child device- blueapi can get the child devices from a string (e.g. "turbo.xfine"), so is this necessary?

Reply:
How are they using the device currently? If the plans are written to take the `turbo_slit` device directly and do the mapping to appropriate Movables does that achieve what they want?

I really think that if you have a device with 3 children, and calling set on that device calls set on exactly 1 of the children with the same value, that's not the correct design for the device.

Reply:
> I really think that if you have a device with 3 children, and calling set on that device calls set on exactly 1 of the children with the same value, that's not the correct design for the device.

please justify the statement. you put forward and abstract argument while the design I implemented reflects the word usage of spectroscopy scientists. they say to set slit to something, not slit xfine during the scan. the other two children are only set during alignment, not scans.

I feel you're saying opinionated things knowing less context than the people who you say are wrong.

Reply:
out of date, applied the fix for float

Reply:
>I really think that if you have a device with 3 children, and calling set on that device calls set on exactly 1 of the children with the same value, that's not the correct design for the device.

I think this is probably one of those things that comes down to taste but wouldn't mind if we had a rule that said we couldn't do it. We do it in MX for a few things where it's very clear from the scientist's perspective that a device only has one thing it can do e.g. we have `Backlight` where we let the user do `backlight.set(IN)`. I think this particular case is less clear but I have no strong opinion.

However, I do think that some of the discussion here was verging on the rude. The purpose of reviews is to try and make the codebase _that we all have some responsibility for_ to be the best it can be and we all ultimately want the same thing. There's no need to get adversarial or defensive.